---
layout: page
title: Publications
permalink: /publicatons/
---



Liu Z\*, Wang L\*, Welch JD\*, Ma H, Zhou Y, Vaseghi HR, Yu S, Wall JB, Alimohamadi S, Zheng M, Yin C, Shen W, Prins JF, Liu J, Qian L. Single-cell transcriptomics reconstructs fate conversion from fibroblast to cardiomyocyte. Nature (02 November 2017). 551, 100–104 doi:10.1038/nature24454 [HTML]
*Equal contribution

Welch JD, Hartemink AJ, Prins JF. MATCHER: manifold alignment reveals correspondence between single cell transcriptome and epigenome dynamics. Genome Biology 2017. 18:138. [PDF]
*Also accepted at RECOMB 2017 under the title “E Pluribus Unum: United States of Single Cells”.

Lackey P, Welch JD, Marzluff WM. TUT7 Catalyzes the Uridylation of the 3’ End of Histone mRNA for Rapid Degradation. RNA 2016. doi:10.1261/rna.058107.116 [PDF]

Welch JD*, Williams LA*, DiSalvo M*, Brandt AT, Marayati R, Sims CE, Allbritton NL, Prins JF, Yeh JJ, Jones CD. Selective Single Cell Isolation for Genomics Using Microraft Arrays. Nucleic Acids Research 2016. doi: 10.1093/nar/gkw700 [PDF]
*Equal contribution

Lyons SM, Cunningham CH, Welch JD, Groh B, Guo AY, Wei B, Whitfield ML, Xiong Y, Marzluff WF. A subset of replication-dependent histone mRNAs are expressed
as polyadenylated RNAs in terminally differentiated tissues. Nucleic Acids Research 2016. doi: 10.1093/nar/gkw620 [PDF]

Welch JD, Hartemink AJ, Prins JF. SLICER: Inferring branched, nonlinear cellular trajectories from single cell RNA-seq data. Genome Biology 2016, 17:106  doi:10.1186/s13059-016-0975-3 [PDF]
*Also accepted at RECOMB 2016 (doi: http://dx.doi.org/10.1101/047845 [PDF])

Welch JD, Hu Y, Prins JF. Robust Detection of Alternative Splicing in a Population of Single Cells. Nucleic Acids Research 2016. doi: 10.1093/nar/gkv1525 [PDF]
*Also accepted as a highlight at ISMB 2016

Brooks L, Lyons SM, Mahoney JM, Welch JD, Liu Z, Marzluff WM, Whitfield ML. A multi-protein occupancy map of the mRNP on the 3′ end of histone mRNAs. RNA 2015. [PDF]

Welch JD*, Slevin MK*, Tatomer D, Duronio RJ, Prins JF, Marzluff WF. EnD-Seq and AppEnD: Sequencing 3′ ends to identify nontemplated tails and degradation intermediates. RNA 2015. [PDF]
*Equal contribution

Welch JD, Baran-Gale J, Perou C, Sethupathy P, Prins JF. Pseudogenes Transcribed in Breast Invasive Carcinoma Show Subtype-Specific Expression and ceRNA Potential. BMC Genomics 2015. [PDF]

Slevin MK, Meaux S, Welch JD, Bigler R, Miliani de Marval PL, Su W, Rhoads RE, Prins JF, Marzluff WF Deep Sequencing Shows Multiple Oligouridylations Are Required for 3′ to 5′ Degradation of Histone mRNAs on Polyribosomes. Molecular cell 53 (6), 1020-1030. March 2014. [HTML]

Lichtenberg J, Kurz K, Liang X, Al-ouran R, Neiman L, Nau LJ, Welch JD, Jacox E, Bitterman T, Ecker K, Elnitski L, Drews F, Lee SS, Welch LR. WordSeeker: concurrent bioinformatics software for discovering genome-wide patterns and
word-based genomic signatures. BMC Bioinformatics. 2010 Dec 21;11 Suppl 12:S6. [PDF]

Lichtenberg J, Yilmaz A, Welch JD, Kurz K, Liang X, Drews F, Ecker K, Lee SS, Geisler M, Grotewold E, Welch LR. The word landscape of the non-coding segments of the Arabidopsis thaliana genome. BMC Genomics. 2009 Oct 8;10:463. [PDF]

Lichtenberg J, Jacox E, Welch JD, Kurz K, Liang X, Yang MQ, Drews F, Ecker K, Lee SS, Elnitski L, Welch LR. Word-based characterization of promoters involved in human DNA repair pathways. BMC Genomics. 2009 Jul 7;10 Suppl 1:S18. [PDF]

Presentations

Welch JD, Hu Y, Prins JF. Robust Detection of Alternative Splicing in a Population of Single Cells. ISMB 2016 (Highlights Track), Orlando, FL.

Welch JD, Hu Y, Prins JF. Robust Detection of Alternative Splicing in a Population of Single Cells. Regulatory Genomics Special Interest Group (RegGen SIG), ISMB 2016, Orlando, FL.

Welch JD, Liu Z, Wang L, Lu J, Lerou P, Purvis J, Qian L, Hartemink A, Prins JF. SLICER: Inferring Branched, Nonlinear Cellular Trajectories from Single Cell RNA-seq Data. RECOMB 2016, Santa Monica, CA.

Welch, JD. Finding Life in High-Dimensional Space: Identifying Cell Types from Single Cell Gene Expression Data. Poster presented at NSF Data Science Workshop 2015, August 2015, Seattle, WA.

Welch JD, Williams LA, DiSalvo M, Sims CE, Allbritton N, Prins JF, Yeh JJ, Jones CD. Selective Single Cell Isolation and RNA Sequencing Using Magnetic Microrafts. Talk presented at RNA 2015, May 2015, Madison, WI.

Welch, JD. Decoding the Symbols of Life: Learning Cell Types and Properties from RNA Sequencing Data. Talk at Dagstuhl Seminar 14381, “Neural-Symbolic Learning and Reasoning”, September 2014. (See Dagstuhl ReportsVolume 4, 2014). [PDF]

Welch JD, Slevin MK, Prins JF, Marzluff W. EnD-seq: A method for finding the 3’ ends of RNA molecules using next-generation sequencing with application to mRNA exonuclease degradation. Poster presented at Symposium on RNA Biology X, North Carolina RNA Society Meeting. October 2013, Chapel Hill, NC.

Welch JD, Prins JF. Investigating the Role of Transcribed Pseudogenes in Breast Cancer. Poster presented at ISMB 2013 and RegGen SIG at ISMB, July 2013, Berlin, Germany.

Students Supervised
Andrew Cowan (SMART Scholar Program)
Bruce Wei (BIOL 395 undergraduate research)
Noah Legall (BIOL 395 undergraduate research, Chancellor’s Science Scholar’s Program)
Richard Clayton (Chancellor’s Science Scholar’s Program)
